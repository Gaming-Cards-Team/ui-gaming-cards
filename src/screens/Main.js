import React from "react";
import Header from "../components/Header";
import Body from "../components/Body";
import Footer from "../components/Footer";

const Main = () => {
  return (
    <div data-testid="mainPage">
      <Header />
      <Body />
      <Footer />
    </div>
  );
};

export default Main;
