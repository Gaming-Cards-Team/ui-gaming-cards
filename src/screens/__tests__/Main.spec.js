import React from 'react';
import renderer from 'react-test-renderer';
import Main from '../Main'; 

describe('Test Main component', () => {

    it('should validate the component', () => {

        const main  = renderer.create(<Main />)
        expect(main).toMatchSnapshot()
    });
});