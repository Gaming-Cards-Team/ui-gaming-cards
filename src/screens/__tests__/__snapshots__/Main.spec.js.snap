// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test Main component should validate the component 1`] = `
Array [
  <nav
<<<<<<< HEAD
    className="navbar navbar-expand navbar-light bg-light"
=======
    className="navbar navbar-expand navbar-light bg-light fixed-top"
>>>>>>> Card-80 [@twjmgorduez] Creates api client
  >
    <a
      className="navbar-brand"
      href="#home"
    >
      <img
        alt="Level Up Logo"
        data-testid="logo"
        height="48"
        src="logo.png"
      />
    </a>
  </nav>,
<<<<<<< HEAD
=======
  <br />,
  " ",
  <br />,
  " ",
  <br />,
>>>>>>> Card-80 [@twjmgorduez] Creates api client
  <img
    alt="Star Wars Banner"
    data-testid="banner"
    src="banner.png"
    width="100%"
  />,
  <div
    style={
      Object {
        "background": "black",
        "color": "white",
        "fontSize": "larger",
        "padding": "10px 30px",
        "width": "100%",
      }
    }
  >
    <div
      className="row"
    >
      <div
        className="col-9"
        style={
          Object {
            "padding": "10px 0px",
          }
        }
      >
        Are you looking for cards for other game cards? check our catalog 
        <a
          href="{#}"
          style={
            Object {
              "color": "#2d963f",
              "fontWeight": "bold",
            }
          }
        >
          here
        </a>
      </div>
      <div
        className="col-3"
        style={
          Object {
            "textAlign": "right",
          }
        }
      >
        <a
          href="https://www.youtube.com/watch?v=3gPu9pp2WOM"
          style={
            Object {
              "color": "#2d963f",
              "fontWeight": "bold",
            }
          }
        >
          How to play 
          <img
            alt="Video Logo"
            data-testid="youtubeVideo"
            height="48"
            src="youtubeVideoImage.png"
          />
        </a>
      </div>
    </div>
  </div>,
  <div
<<<<<<< HEAD
    style={
      Object {
        "paddingBottom": 50,
        "paddingLeft": 100,
        "paddingRight": 100,
        "paddingTop": 30,
      }
    }
  >
    <div
      className="card-deck"
    >
      <div
        className="col-lg-3 col-md-4 col-sm-6 col-12"
      >
        <div
          style={
            Object {
              "paddingBottom": 15,
              "paddingTop": 10,
            }
          }
        >
          <div
            className="card"
          >
            <div
              className="card-body"
            >
              <img
                className="card-img-top"
                src="https://swdestinydb.com/bundles/cards/en/01/01001.jpg"
              />
            </div>
          </div>
        </div>
      </div>
      <div
        className="col-lg-3 col-md-4 col-sm-6 col-12"
      >
        <div
          style={
            Object {
              "paddingBottom": 15,
              "paddingTop": 10,
            }
          }
        >
          <div
            className="card"
          >
            <div
              className="card-body"
            >
              <img
                className="card-img-top"
                src="https://swdestinydb.com/bundles/cards/en/01/01002.jpg"
              />
            </div>
          </div>
        </div>
      </div>
      <div
        className="col-lg-3 col-md-4 col-sm-6 col-12"
      >
        <div
          style={
            Object {
              "paddingBottom": 15,
              "paddingTop": 10,
            }
          }
        >
          <div
            className="card"
          >
            <div
              className="card-body"
            >
              <img
                className="card-img-top"
                src="https://swdestinydb.com/bundles/cards/en/01/01003.jpg"
              />
            </div>
          </div>
        </div>
      </div>
      <div
        className="col-lg-3 col-md-4 col-sm-6 col-12"
      >
        <div
          style={
            Object {
              "paddingBottom": 15,
              "paddingTop": 10,
            }
          }
        >
          <div
            className="card"
          >
            <div
              className="card-body"
            >
              <img
                className="card-img-top"
                src="https://swdestinydb.com/bundles/cards/en/01/01004.jpg"
              />
            </div>
          </div>
        </div>
      </div>
      <div
        className="col-lg-3 col-md-4 col-sm-6 col-12"
      >
        <div
          style={
            Object {
              "paddingBottom": 15,
              "paddingTop": 10,
            }
          }
        >
          <div
            className="card"
          >
            <div
              className="card-body"
            >
              <img
                className="card-img-top"
                src="https://swdestinydb.com/bundles/cards/en/01/01005.jpg"
              />
            </div>
          </div>
        </div>
      </div>
      <div
        className="col-lg-3 col-md-4 col-sm-6 col-12"
      >
        <div
          style={
            Object {
              "paddingBottom": 15,
              "paddingTop": 10,
            }
          }
        >
          <div
            className="card"
          >
            <div
              className="card-body"
            >
              <img
                className="card-img-top"
                src="https://swdestinydb.com/bundles/cards/en/01/01006.jpg"
              />
            </div>
          </div>
        </div>
      </div>
      <div
        className="col-lg-3 col-md-4 col-sm-6 col-12"
      >
        <div
          style={
            Object {
              "paddingBottom": 15,
              "paddingTop": 10,
            }
          }
        >
          <div
            className="card"
          >
            <div
              className="card-body"
            >
              <img
                className="card-img-top"
                src="https://swdestinydb.com/bundles/cards/en/01/01007.jpg"
              />
            </div>
          </div>
        </div>
      </div>
      <div
        className="col-lg-3 col-md-4 col-sm-6 col-12"
      >
        <div
          style={
            Object {
              "paddingBottom": 15,
              "paddingTop": 10,
            }
          }
        >
          <div
            className="card"
          >
            <div
              className="card-body"
            >
              <img
                className="card-img-top"
                src="https://swdestinydb.com/bundles/cards/en/01/01008.jpg"
              />
            </div>
          </div>
        </div>
      </div>
      <div
        className="col-lg-3 col-md-4 col-sm-6 col-12"
      >
        <div
          style={
            Object {
              "paddingBottom": 15,
              "paddingTop": 10,
            }
          }
        >
          <div
            className="card"
          >
            <div
              className="card-body"
            >
              <img
                className="card-img-top"
                src="https://swdestinydb.com/bundles/cards/en/01/01009.jpg"
              />
            </div>
          </div>
        </div>
      </div>
      <div
        className="col-lg-3 col-md-4 col-sm-6 col-12"
      >
        <div
          style={
            Object {
              "paddingBottom": 15,
              "paddingTop": 10,
            }
          }
        >
          <div
            className="card"
          >
            <div
              className="card-body"
            >
              <img
                className="card-img-top"
                src="https://swdestinydb.com/bundles/cards/en/01/01010.jpg"
              />
            </div>
          </div>
        </div>
      </div>
      <div
        className="col-lg-3 col-md-4 col-sm-6 col-12"
      >
        <div
          style={
            Object {
              "paddingBottom": 15,
              "paddingTop": 10,
            }
          }
        >
          <div
            className="card"
          >
            <div
              className="card-body"
            >
              <img
                className="card-img-top"
                src="https://swdestinydb.com/bundles/cards/en/01/01011.jpg"
              />
            </div>
          </div>
        </div>
      </div>
      <div
        className="col-lg-3 col-md-4 col-sm-6 col-12"
      >
        <div
          style={
            Object {
              "paddingBottom": 15,
              "paddingTop": 10,
            }
          }
        >
          <div
            className="card"
          >
            <div
              className="card-body"
            >
              <img
                className="card-img-top"
                src="https://swdestinydb.com/bundles/cards/en/01/01012.jpg"
              />
            </div>
          </div>
        </div>
      </div>
    </div>
=======
    className=""
    style={Object {}}
  >
    <div
      style={
        Object {
          "paddingBottom": 50,
          "paddingLeft": 100,
          "paddingRight": 100,
          "paddingTop": 30,
        }
      }
    >
      <div
        className="card-deck"
      />
    </div>
    <div
      style={
        Object {
          "textAlign": "center",
        }
      }
    >
      Loading...
    </div>
    <div
      id="11588137155162_scrollToFetch_bottom"
    />
>>>>>>> Card-80 [@twjmgorduez] Creates api client
  </div>,
  <nav
    className="navbar navbar-expand navbar-light fixed-bottom"
    style={
      Object {
        "backgroundColor": "#2d963f",
      }
    }
  >
    <div
      data-testid="footer-text"
      style={
        Object {
          "color": "white",
          "textAlign": "center",
          "width": "100%",
        }
      }
    >
      Copyright Star Wars and all associated names and/or images are copyright Lucasfilm Ltd.
    </div>
  </nav>,
]
`;
